assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -158,6 +158,9 @@ che.workspace.startup_debug_log_limit_bytes=10485760
 # This configuration is mainly required for workspace idling when the OpenShift OAuth is enabled.
 che.workspace.stop.role.enabled=true
 
+#Specifies whether che is deployed with devworkspaces enabled.
```

Comment:
I would possibly add more detail to this, so that the users don't expect devworkspace support to magically appear by just flipping this to "true".

Something along the lines of:

```
This property is set by the Che operator if it also installed the support for DevWorkspaces. This property is used
to advertise this fact to the Che dashboard. It does not make sense to change the value of this property manually.
```